cellprofiler/pipeline.py

```diff
@@ -951,6 +951,12 @@ def rl():
                     ).format(git_hash, pipeline_date)
                     logging.warning(message)
                 else:
+                    # pipeline versions pre-3.0.0 have unpredictable formatting
+                    if pipeline_version == 300:
```

Comment:
Yes, but it wouldn’t work with bundled applications since it assumes the package (and version information) is available from the Python path.